### ISSUE TITLE
Static/build artifact create project command

### ISSUE THREAD

**Issue by ataylorme** [pantheon-systems/terminus-build-tools-plugin#237]
- Gatsby deployed with CI proof of concept
- One Pantheon site for WordPress/Drupal backend
- Another Pantheon site for the build artifact

cc @populist

**Comment by greg-1-anderson** [comment]
Cool idea. Has anyone made a Gatsby / Drupal or Gatsby / WordPress POC yet?

**Comment by ataylorme** [comment]
> Has anyone made a Gatsby / Drupal or Gatsby / WordPress POC yet?

Yes, Matt and I made one the last few days with Gatsby consuming a Drupal 8 site. 

We also started discussing tightening it up and hosting the front-end on Pantheon then working backward to define the steps needed to set up a similar project automatically with Build Tools as a Q4 initiative. Product is very interested in a headless proof of concept and helping to identify the gaps to an ideal dashboard based workflow.

**Comment by ataylorme** [comment]
@populist want to schedule some time in September for hacking on this?

**Comment by populist** [comment]
Absolutely. Loving that POC energy.

**Comment by ataylorme** [comment]
Summary of the work to be done:
- Modify `build:project:create` to accept `gatsby` as an option with a new flag for `--cms`, defaulting to `wp`
  - Create 2 GitHub projects, one for the CMS and one for Gatsby
  - The CMS project deploys to Pantheon with the same functionality as exists in Build Tools now
  - The Gatsby project is modeled from [the Gatsby WordPress starter](https://github.com/GatsbyCentral/gatsby-starter-wordpress), built with GitHub actions and deployed to GitHub pages

Out of scope for the proof of concept:
- Support for providers other than GitHub
- WebHooks to re-build Gatsby on CMS content changes or code deployments
- Drupal example
- Accepting a `--git` parameter other than `github`
- Accepting a `--ci` parameter other than `circleci`
